xclient_gui/desktop/views/components/chat.py

```diff
@@ -1,48 +1,38 @@
-from PyQt5.QtWidgets import QVBoxLayout, QWidget, QFrame
+from PyQt5.QtWidgets import QListWidget, QWidget, QVBoxLayout, QLineEdit, QHBoxLayout, QPushButton, \
+    QListWidgetItem
 
-from xclient_gui.desktop.views.components.tree import Tree
 
-
-class Chat(QWidget):
+class MessagesList(QListWidget):
     def __init__(self):
         super().__init__()
+        self.setStyleSheet('background-color:gray')
+        self.setWordWrap(True)
+        self.show()
 
-        # self.addItem(QLayoutItem())
-        # self.addItem("Item 2")
-        # self.addItem("Item 3")
-        # self.addItem("Item 4")
-        # self.addItem("Item 5")
-        # self.setView(QListView())
-        # box = QVBoxLayout
-
-        #
-
-        hbox = QVBoxLayout()
-
-        tree = Tree()
-        tree.setFrameShape(QFrame.StyledPanel)
-        tree.setStyleSheet('background-color:white')
 
-        tree2 = Tree()
-        tree2.setFrameShape(QFrame.StyledPanel)
-        tree2.setStyleSheet('background-color:red')
+class NewMessageInput(QHBoxLayout):
+    def __init__(self):
+        super().__init__()
 
-        # chat = Chat()
-        # chat.setStyleSheet('background-color:green')
+        message_input = QLineEdit()
```

Comment:
Consider replacing QLineEdit to multiline QPlainTextEdit

Reply:
I will try it. A possible change will appear in the future pull request.